selfdrive/car/hyundai/values.py

```diff
@@ -52,99 +52,46 @@ def __init__(self, CP):
-  SONATA_LF = "HYUNDAI SONATA 2019"
-  STARIA_4TH_GEN = "HYUNDAI STARIA 4TH GEN"
-  TUCSON = "HYUNDAI TUCSON 2019"
-  PALISADE = "HYUNDAI PALISADE 2020"
-  VELOSTER = "HYUNDAI VELOSTER 2019"
-  SONATA_HYBRID = "HYUNDAI SONATA HYBRID 2021"
-  IONIQ_5 = "HYUNDAI IONIQ 5 2022"
-  IONIQ_6 = "HYUNDAI IONIQ 6 2023"
-  TUCSON_4TH_GEN = "HYUNDAI TUCSON 4TH GEN"
-  SANTA_CRUZ_1ST_GEN = "HYUNDAI SANTA CRUZ 1ST GEN"
-  CUSTIN_1ST_GEN = "HYUNDAI CUSTIN 1ST GEN"
+  # Static Flags
 
-  # Kia
-  KIA_FORTE = "KIA FORTE E 2018 & GT 2021"
-  KIA_K5_2021 = "KIA K5 2021"
-  KIA_K5_HEV_2020 = "KIA K5 HYBRID 2020"
-  KIA_K8_HEV_1ST_GEN = "KIA K8 HYBRID 1ST GEN"
-  KIA_NIRO_EV = "KIA NIRO EV 2020"
-  KIA_NIRO_EV_2ND_GEN = "KIA NIRO EV 2ND GEN"
-  KIA_NIRO_PHEV = "KIA NIRO HYBRID 2019"
-  KIA_NIRO_PHEV_2022 = "KIA NIRO PLUG-IN HYBRID 2022"
-  KIA_NIRO_HEV_2021 = "KIA NIRO HYBRID 2021"
-  KIA_NIRO_HEV_2ND_GEN = "KIA NIRO HYBRID 2ND GEN"
-  KIA_OPTIMA_G4 = "KIA OPTIMA 4TH GEN"
-  KIA_OPTIMA_G4_FL = "KIA OPTIMA 4TH GEN FACELIFT"
-  KIA_OPTIMA_H = "KIA OPTIMA HYBRID 2017 & SPORTS 2019"
-  KIA_OPTIMA_H_G4_FL = "KIA OPTIMA HYBRID 4TH GEN FACELIFT"
-  KIA_SELTOS = "KIA SELTOS 2021"
-  KIA_SPORTAGE_5TH_GEN = "KIA SPORTAGE 5TH GEN"
-  KIA_SORENTO = "KIA SORENTO GT LINE 2018"
-  KIA_SORENTO_4TH_GEN = "KIA SORENTO 4TH GEN"
-  KIA_SORENTO_HEV_4TH_GEN = "KIA SORENTO HYBRID 4TH GEN"
-  KIA_STINGER = "KIA STINGER GT2 2018"
-  KIA_STINGER_2022 = "KIA STINGER 2022"
-  KIA_CEED = "KIA CEED INTRO ED 2019"
-  KIA_EV6 = "KIA EV6 2022"
-  KIA_CARNIVAL_4TH_GEN = "KIA CARNIVAL 4TH GEN"
+  # If 0x500 is present on bus 1 it probably has a Mando radar outputting radar points.
+  # If no points are outputted by default it might be possible to turn it on using  selfdrive/debug/hyundai_enable_radar_points.py
+  MANDO_RADAR = 2**12
+  CANFD = 2**13
 
-  # Genesis
-  GENESIS_GV60_EV_1ST_GEN = "GENESIS GV60 ELECTRIC 1ST GEN"
-  GENESIS_G70 = "GENESIS G70 2018"
-  GENESIS_G70_2020 = "GENESIS G70 2020"
-  GENESIS_GV70_1ST_GEN = "GENESIS GV70 1ST GEN"
-  GENESIS_G80 = "GENESIS G80 2017"
-  GENESIS_G90 = "GENESIS G90 2017"
-  GENESIS_GV80 = "GENESIS GV80 2023"
+  # The radar does SCC on these cars when HDA I, rather than the camera
+  RADAR_SCC = 2**14
+  CAMERA_SCC = 2**15
+  CHECKSUM_CRC8 = 2**16
+  CHECKSUM_6B = 2**17
 
+  # these cars require a special panda safety mode due to missing counters and checksums in the messages
+  LEGACY = 2**18
 
-class Footnote(Enum):
-  CANFD = CarFootnote(
-    "Requires a <a href=\"https://comma.ai/shop/can-fd-panda-kit\" target=\"_blank\">CAN FD panda kit</a> if not using " +
-    "comma 3X for this <a href=\"https://en.wikipedia.org/wiki/CAN_FD\" target=\"_blank\">CAN FD car</a>.",
-    Column.MODEL, shop_footnote=False)
+  # these cars have not been verified to work with longitudinal yet - radar disable, sending correct messages, etc.
+  UNSUPPORTED_LONGITUDINAL = 2**19
+
+  CANFD_NO_RADAR_DISABLE = 2**20
+
+  CLUSTER_GEARS = 2**21
+  TCU_GEARS = 2**22
```

Comment:
`2**20` is much easier to understand than `1048576`

Reply:
hopefully we don't have to worry about whether our system is 32 bits or 64

Reply:
We should probably have a test warning us of this, we're pretty close to 32 bits